### PR TITLE
TM-4110, AIM only show active remarks in glossary

### PR DESCRIPTION
**Pull request by M00staff**
Dual Merge: 
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/356)

Not sure if the mock code is worth merging or not - but that's how i tested it. 

https://metaphase.atlassian.net/browse/TM-4110

### CODE REVIEW

File: src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx [hunk 1]
```diff
@@ -46,14 +46,14 @@ const AgendaItemResearchPane = forwardRef((props = { perdet: '', clientData: {},
   const classificationsProps = { classifications, clientClassifications };
 
   const { data, error, loading } = useDataLoader(api().get, `/fsbid/assignment_history/${perdet}/`);
-  const remarks = useDataLoader(api().get, '/fsbid/agenda/remarks/');
```
Comment (M00staff) [inline src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:49]: so, this `useDataLoader` hook is nifty - it works like the Apollo Client `useQuery`, which I'm a big fan of... 

the catch seems to be that we can't destructure anything nested out of it, so something like:
`const { data : { data : remarks } = {} } = useDataLoader(api().get, '/fsbid/agenda/remarks/');`
`const remarks_data = remarks?.results || [];` 

is what i'd prefer (or some flavor of it) - i can see a little more why `lodash.get` is preferred 😅

**Comment by elizabeth-jimenez** [comment]
Was this the ticket we said we wanted to discuss further as a team at internal Grooming?

**Comment by M00staff** [comment]
> Was this the ticket we said we wanted to discuss further as a team at internal Grooming?

doesn't ring a bell - but, we can chat about it tomorrow

File: src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx [hunk 2]
```diff
@@ -46,14 +46,14 @@ const AgendaItemResearchPane = forwardRef((props = { perdet: '', clientData: {},
...
+  const { data: remarks } = useDataLoader(api().get, '/fsbid/agenda/remarks/');
```
Comment (pveland621) [inline src/Components/Agenda/AgendaItemResearchPane/AgendaItemResearchPane.jsx:49]: just a heads up as we might run into merge conflicts, my TM-4016 touches this code to have individual data, error, and loading variables for the tabs on the AIM research pane

image

**Review by elizabeth-jimenez — APPROVED** [review]
✅❎

**Pull request merged**